src/components/TeamMember/TeamMember.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import SocialLink,{socialClasses} from '../SocialLink';
+import './TeamMember.css';
+import placeholder from './img/placeholder.png';
+
+function TeamMember({ firstName, lastName, photoUrl, info, links }) {
+    const parseLinks = (linkElement) => {
+        let linkType = '';
+
+        if(typeof(linkElement) !== "object") {
+            for(let type of Object.keys(socialClasses))
```

Comment:
Dodałbym klamry, bo takie dwa jednolinijkowce w jednym to może stworzyć jakiś problem przy developmencie